README.md

```
@@ -1,6 +1,115 @@
+
+### Environment Variables
+
+| Variable | Default | Description |
+|----------|---------|-------------|
+| `BA_DEBUG` | `false` | Enable debug mode |
+| `BA_LOG_LEVEL` | `INFO` | Logging level |
+| `BA_CORS_ORIGINS` | `["*"]` | Allowed CORS origins |
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)